content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,246 @@
+   since the :guilabel:`Source` of the stock move is a :abbr:`PO (Purchase Order)`.
+
+   The :guilabel:`Reference` operation, `WH/IN/00015`, which is a warehouse reception of products
+   from the vendor, would further explain why the apples were moved to the wrong location.
+
+   .. image:: putaway/stock-move-report.png
+      :align: center
+      :alt: Open stock move report
+
+To follow an operation identified from the :guilabel:`Product Moves` report, navigate to
+:menuselection:`Inventory --> Operations --> Transfers`. This is the page that displays all
+warehouse transfers-- internal or external. Search for and click on the corresponding
+:guilabel:`Reference` operation to view the document explaining why the product was moved to the
+wrong location.
+
+.. tip::
+   When there are too many records, use either the :guilabel:`Filters` or :guilabel:`Group by`
+   features to sort and :ref:`find desired records
+   <inventory/routes/strategies/putaway/use-advanced-filters>`. It is useful to search exclusively
+   for the receipt operation type because putaway rules store products *into* a specific stock
+   location.
+
+.. example::
+   Continuing to diagnose why the product, `Apple`, was placed in `WH/Stock/Shelf 1`, navigate to
+   the list of all transfers in :menuselection:`Inventory --> Operations --> Transfers`. Click on
+   the list item where the :guilabel:`Reference` field matches the `WH/IN/00015` operation
+   identified in the :ref:`previous example <inventory/strategies/putaway/stock-moves>` to open the
+   receipt.
+
+   .. image:: putaway/transfers.png
+      :align: center
+      :alt: In the Transfers page in the Inventory app, select the desired transfer to view the
+            record.
+
+   This receipt describes which products were received from the supplier, with the affiliated
+   :abbr:`PO (Purchase Order)` linked in the :guilabel:`Source Document` field. On the warehouse
+   reception document, the field :guilabel:`Destination Location` shows where the product is
+   received upon entering the warehouse.
+
+   Clicking on the :guilabel:`≣ (Detailed Operations)` icon to the right of the product line opens a
+   window that shows the product was moved to the wrong location. Based on this information it can
+   be gathered that the putaway rule was triggered but the location was set incorrectly. Rectify
+   this in :menuselection:`Inventory --> Configuration --> Putaway Rules`.
+
+   .. image:: putaway/detailed-operations.png
+      :align: center
+      :alt: Click Detailed Operations button on the right of warehouse transfers page to the right
+            of the WH reception document.
+
+   .. image:: putaway/automated.png
+      :align: center
+      :alt: Show putaway rule in detailed operations popup.
+
+.. _inventory/routes/strategies/putaway/use-advanced-filters:
+
+Advanced filter search
+----------------------
+
+Keep track of and trace products on the :guilabel:`Product Moves` page. Navigate to the page from
```

Comment:
page --> report
doing this also cuts out some words and creates a more succinct compound sentence
```suggestion
Keep track of and trace products on the :guilabel:`Product Moves` report by navigating to
```